apps/web/src/components/Table/ClientManagedTable/tableClassificationPills.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { notEmpty } from "@gc-digital-talent/helpers";
+import { Pill } from "@gc-digital-talent/ui";
+import { Maybe } from "@gc-digital-talent/graphql";
+import { Classification } from "~/api/generated";
+import uniqueId from "lodash/uniqueId";
+
+interface TableClassificationPillsProps {
+  classifications: Maybe<Maybe<Classification>[]>;
+}
+
+const TableClassificationPills = ({
+  classifications,
+}: TableClassificationPillsProps) => {
+  const filteredClassifications = classifications
+    ? classifications.filter(notEmpty)
+    : [];
+  const pillsArray = filteredClassifications.map((classification, index) => {
+    return (
+      <span
+        key={uniqueId()}
```

Comment:
Can we find a better key?  How about classification.id?

Reply:
Lol forsure (not sure why I did that) 😆 Fixed by [d8b9e80](https://github.com/GCTC-NTGC/gc-digital-talent/pull/7101/commits/d8b9e80bcd9916f4f1c6eabe2c05124fdd352c9b)